From: Jeroen Dekkers <jeroen@dekkers.ch>
Date: Tue, 23 Oct 2012 16:51:45 +0200
Subject: Detect Mac OS X 10.8 Address Book and iCal correctly.

Origin: upstream
---
 sope-appserver/NGObjWeb/WEClientCapabilities.m |    7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/sope-appserver/NGObjWeb/WEClientCapabilities.m b/sope-appserver/NGObjWeb/WEClientCapabilities.m
index 397ec27..d6fb7ef 100644
--- a/sope-appserver/NGObjWeb/WEClientCapabilities.m
+++ b/sope-appserver/NGObjWeb/WEClientCapabilities.m
@@ -329,8 +329,11 @@
     /* Wget */
     self->browser = WEUA_Wget;
   }
-  else if (strstr(ua, "DAVAccess") || strstr(ua, "CardDAVPlugin")
-           || strstr(ua, "CalendarStore") || strstr(ua, "CoreDAV/")) {
+  else if (strstr(ua, "DAVAccess")
+           || strstr(ua, "CardDAVPlugin")
+           || strstr(ua, "CalendarStore")
+           || strstr(ua, "CoreDAV/")
+           || (strstr(ua, "AddressBook") || strstr(ua, "Calendar")) && strstr(ua, "Mac OS X")) {
     /* Apple MacOSX 10.2.1 / iCal 1.0 DAV Access Framework */
     self->browser = WEUA_AppleDAVAccess;
   }
